### ISSUE TITLE
Two kind requests

### ISSUE THREAD

**Issue by mangoose20026** [Bolls-Bible/bain#22]
1. Add support as well as a new page for tamil bible like English (https://bolls.life/) to search and read online. Tamil bible is available in all formats whichever you like to use (https://github.com/godlytalias/Bible-Database/tree/master/Tamil)
2. Add support to import sermon text files to search like how we search the bible. That would be really helpful. (For example, import spurgeon sermon text files, import John Bunyan sermons files, import Calvin Sermon files)

**Comment by Bohooslav** [comment]
1. We already have Tamil translation https://bolls.life/TBSI/1/1/. To switch translation click on translation abbreviation at the books menu and pick new translations. You may also search for translations in main search suggestions. If you desire some other translation let me know.
2. Sermons and third party texts are against our policy of «Sola Scriptura». It wasn't designed to have those.

**Comment by mangoose20026** [comment]
#2 - is it possible to load temporarily and do the search ? browser based - once browser closed, the data deleted....just push to indexed db or have it in cache ?

**Comment by Bohooslav** [comment]
Sorry, I am not sure if I understood you. What do you mean?

**Comment by mangoose20026** [comment]
I meant is it possible to provide a new page for just import sermons and do a full content search.....

Like Bible -->

Have thousands of sermons. Each Sermon will have paragraphs

So bible chapter title should be sermon title
Bible chapter would be each para in sermons. Example Genesis 1,2...50 --> Sermon1 - 1,2....100 paragraphs or no of paragraphs  in sermon

(or) import just raw text sermons page to do full content search

basically i would like to do sermons and bible via the same page navigation.

i can place the sermons in respective format if needed

**Comment by Bohooslav** [comment]
Each translation differs, chapters may not match, how are you gonna cross match chapters and sermons through different translations? Also, how the UI would look like? Sermon paragraphs in between Bible text? Clunky! Also, what about search, how should I handle search? There are other software products that provide sermons and stuff, I desire to keep Bolls simple and clean, «Sola Scriptura» is the motto. Many people use Bolls in part because it is not flooded with sermons or converted into social app or advertisements space. I want it to remain confession agnostic, and while yes I may include confession specific translations of Bible, but I don't feel comfortable to push through my software some church sermons and theology. Sorry, but I think you must come up with some third party solution.
